src/loader.c

```diff
@@ -134,6 +135,7 @@ static const image_decoder decoders[] = {
     &LOADER_FUNCTION(tiff),
 #endif
     &LOADER_FUNCTION(tga),
+    &LOADER_FUNCTION(qoi),
```

Comment:
The TGA format does not have any "magic" header, so it would be better to process it last.
Please, swap qoi and tga functions in this array.